src/front-end/typescript/lib/pages/dashboard.tsx

```diff
@@ -147,53 +148,43 @@ function makeVendorBodyRows(
 function makePublicSectorBodyRows(
   cwu: CWUO.CWUOpportunitySlim[],
   swu: SWUO.SWUOpportunitySlim[],
+  twu: TWUO.TWUOpportunitySlim[],
   viewerUser: User
 ): Table.BodyRows {
   return [
     ...cwu.map((o) => adt("cwu" as const, o)),
-    ...swu.map((o) => adt("swu" as const, o))
+    ...swu.map((o) => adt("swu" as const, o)),
+    ...twu.map((o) => adt("twu" as const, o))
   ]
     .filter((o) => o.value.createdBy?.id === viewerUser.id)
     .sort((a, b) => compareDates(a.value.createdAt, b.value.createdAt) * -1)
     .map((p) => {
-      const defaultTitle =
-        p.tag === "cwu"
-          ? CWUO.DEFAULT_OPPORTUNITY_TITLE
-          : SWUO.DEFAULT_OPPORTUNITY_TITLE;
+      const defaultTitle = oppHelpers(p).dashboard.getDefaultTitle();
       return [
         {
           children: (
             <div>
               <Link
                 dest={routeDest(
-                  adt(
-                    p.tag === "cwu"
-                      ? "opportunityCWUEdit"
-                      : "opportunitySWUEdit",
-                    { opportunityId: p.value.id }
-                  )
+                  oppHelpers(p).dashboard.getOppEditRoute(p.value.id)
                 )}>
                 {p.value.title || defaultTitle}
               </Link>
               <div className="small text-secondary text-uppercase">
-                {p.tag === "cwu" ? "Code With Us" : "Sprint With Us"}
+                {p.tag === "cwu"
+                  ? "Code With Us"
+                  : p.tag === "swu"
+                  ? "Sprint With Us"
+                  : "Team With Us"}
```

Comment:
Yeah, definitely, was just thinking about where to put it in the meantime. Since that's used in multiple places I was thinking of a ProgramType interface (as opposed to sticking it in with the other opportunity helpers)